content/articles/getting-started-clickfunnels.markdown

```diff
@@ -0,0 +1,222 @@
+---
+title: Getting Started With Your DNSimple Account After Transferring From ClickFunnels
+excerpt: A collection of articles to help you get started with your new DNSimple account quickly.
+categories:
+- DNSimple
+---
+
+# Getting Started With Your DNSimple Account After Transferring From ClickFunnels
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Welcome to [DNSimple](/articles/dnsimple-services/)! We are going to make your transition from ClickFunnels as simple as possible. [Simple](/articles/how-is-dnsimple-pronounced/) is in our name, and we're not going to let you down.
+
+## Step 1: Setting up a DNSimple account and trial
+
+<div class="section-steps" markdown="1">
+1. Go to [DNSimple.com](https://dnsimple.com). In the upper right corner of the screen, click the button that says <label>Get Started</label>.
+1. Complete the form using the **same email address that is associated with your ClickFunnels account**. Enter a password and confirm it. Click <label>Join DNSimple</label>.
+1. Now choose a plan. Select <label>Continue</label> under <label>Individual</label>, then choose <label>Get Started</label> under <label>Personal</label> or <label>Professional</label>, depending on your needs.
+
+    <tip>
+    [Learn the differences](/articles/personal-vs-pro/) between the Personal plan and the Professional plan.
+    </tip>
+
+1. Add your credit card details then click <label>Start my free trial</label>.
+
+    <info>
+    You can cancel at any time during your 30 day trial without being charged. If you choose to purchase products such as another domain or an SSL certificate during the free trial, your card will be charged. Read more about [account activation](/articles/account-activation/).
```

Comment:
Let's include in the explanation for the credit card requirement that we ask for the card to set up your account and also to cover charges outside of the subscription. It isn't explicit enough as to why we need the card during a trial. Also we should put that the account will not be activated until the card has been added